src/coreclr/pal/src/include/pal/dbgmsg.h

```diff
@@ -259,9 +259,15 @@ extern Volatile<BOOL> dbg_master_switch ;
     static const DBG_CHANNEL_ID defdbgchan = DCI_##x
 
 /* Is debug output enabled for the given level and channel? */
+#ifdef __wasm__
```

Comment:
Is there no equivalent to env vars on WASM?

Reply:
the emscripten has virtualised environment, we can leverage that (https://emscripten.org/docs/porting/connecting_cpp_and_javascript/Interacting-with-code.html#environment-variables)